### PR TITLE
[WIFI-5976] integrate Quali advanced blueprint into work

### PR DESCRIPTION
**Pull request by oblom0v**
Signed-off-by: Johann Hoffmann <johann.hoffmann@mailbox.org>

### CODE REVIEW

**Review by dunaev-opsfleet — APPROVED** [review]

File: .github/workflows/advanced.yml [hunk 1]
```diff
@@ -5,6 +5,38 @@ env:
 
 on:
   workflow_dispatch:
+    inputs:
+      openwifi_revision:
+        required: true
+        default: 'main'
+        description: 'revision of the Open Wifi Helm chart'
+      openwifi_gw_revision:
```
Comment (brennerm) [inline .github/workflows/advanced.yml:13]: Think we can remove the sub chart versions on request by Jaspreet.

Comment (oblom0v) [reply .github/workflows/advanced.yml:13]: Yep, from what I understand the changes to the deploy script have to be merged first + the blueprint has to be adapted.

File: .github/workflows/advanced.yml [hunk 2]
```diff
@@ -13,6 +45,46 @@ defaults:
...
+      openwifi_ui_revision: ${{ steps.vars.outputs.ui}}
+      ap_models: ${{ steps.vars.outputs.ap_models}}
+      wifi_type: ${{ steps.vars.outputs.wifi_type}}
+      marker_expression: ${{ steps.vars.outputs.marker_expression }}
+
+    steps:
+      - name: set variables
+        id: vars
+        run: |
+          echo ::set-output name=openwifi::$(echo "${{ github.event.inputs.openwifi_revision || 'v2.4.0-RC2' }}")
```
Comment (brennerm) [inline .github/workflows/advanced.yml:64]: Think we should default to the latest branches here.

File: .github/workflows/advanced.yml (hunk 1 above)
Comment (brennerm) [reply .github/workflows/advanced.yml:13]: Don't think so as the Python script will already set some default values, right?

Comment (oblom0v) [reply .github/workflows/advanced.yml:13]: yes, depends a bit on whether the ability of testing specific chart versions is required until the changes. If I get it correctly the default values would just overwrite the subchart values every time with master/main regardless of what has been set as the `openwifi_revision` input.

**Pull request merged**